ingestors/kafka/src/main/scala/hydra/kafka/algebras/KafkaAdminAlgebra.scala

```diff
@@ -226,7 +226,11 @@ object KafkaAdminAlgebra {
         ref.get.map(_.get(name))
 
       override def getTopicNames: F[List[TopicName]] =
-        ref.get.map(_.keys.toList)
+        ref.get.map(
+          k => {
+            k.keys.toList
+          }
+        )
```

Comment:
Looks like this can be reverted
```suggestion
        ref.get.map(_.keys.toList)
```